addons/mail/static/src/messaging/component/attachment/attachment_tests.js

```diff
@@ -484,14 +484,14 @@ QUnit.test('auto layout with image', async function (assert) {
             return this._super(...arguments);
         },
     });
-    const attachmentLocalId = this.env.store.dispatch('createAttachment', {
+    const attachment = this.env.entities.Attachment.create({
```

Comment:
don't think this would though, because we need start before entity, and we need entity before component. A single helper would just make it hard to read (params with lots of nested objects), and have to handle requirements among them... callbacks in params? No, I think it's best always create env (`start()`), then create entity, then create component. Much easier to understand what the test does, even if setup requires some amount of lines (which btw, with a helper we may just gain one line or 2)